formats/protobuf/src/main/java/io/cloudevents/protobuf/ProtoSupport.java

```diff
@@ -0,0 +1,30 @@
+package io.cloudevents.protobuf;
+
+/**
+ * General support functions.
+ */
+
+class ProtoSupport {
```

Comment:
nit, this class can be final
```suggestion
final class ProtoSupport {
```

Reply:
ack